core-bundle/contao/languages/en/default.xlf

```diff
@@ -305,6 +305,9 @@
       <trans-unit id="ERR.passkeyAttestationFailure">
         <source>Passkey could not be created.</source>
       </trans-unit>
+      <trans-unit id="ERR.passkeyInvalidStateError">
+        <source>Authenticator was already registered by user.</source>
+      </trans-unit>
```

Comment:
Where is this used?

Reply:
This was present in an earlier version of this PR but I have since replaced the error message with the more general "Passkey could not be created.". I will remove the translation.